### ISSUE TITLE
Index: link to hero tour is broken

### ISSUE THREAD

**Issue by gaglage** [angular/angular-cli#6694]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.

Install angular-cli and check http://localhost:4200/

### Mention any other details that might be useful.

A minor bug: link to hero tour is broken in index item since angular.io update
http://angularjs.blogspot.com.es/

**Comment by Brocco** [comment]
Thanks for reporting this... the line that will need fixed is located [here](https://github.com/angular/angular-cli/blob/e87598e5db0229f3464b0c40d4f87322a0b76db7/packages/%40angular/cli/blueprints/ng/files/__path__/app/app.component.html#L11) with the correct link:

https://angular.io/tutorial

**Comment by jiverson** [comment]
Looks like this is fixed #6672
